sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 #### Breaking Changes
 
 #### Bugs Fixed
+* Added improvement in `RntbdConnectionStateListener` to better handling scenarios when connection is closed unexpectedly - See [PR 32936](https://github.com/Azure/azure-sdk-for-java/pull/32936)
```

Comment:
How about: All connection closures will result in unhealthy status?

Reply:
hmm, what about `Added better handling for connection being closed unexpectedly` - trying to avoid mention the replica status as it is implementation details and thinking about changing the name